web/templates/note/edit.html.eex

```diff
@@ -0,0 +1,5 @@
+<h1>Edit User</h1>
+
+<%= render "form.html", changeset: @changeset, action: note_path(@conn, :update, @note) %>
+
+<%= link "Back", to: note_path(@conn, :index) %>
```

Comment:
I don't really like `Back`. The browser already has a back button, and we don't want to reimplement that. I would be down with a `Cancel` button though (I realize this is just a cosmetic/name change).

In which case, a cancel button seems like a property of the form. Maybe move it into the form template?